web/src/edit/NeighbourhoodMode.svelte

```diff
@@ -332,126 +330,157 @@
     {/if}
 
     <h2>Editing tools</h2>
-    <div class="tool-palette">
-      <button
-        on:click={() => (action = { kind: "filter" })}
-        disabled={action.kind == "filter"}
-        class:active={action.kind == "filter"}
-        class:outline={action.kind != "filter"}
-        data-tooltip="Add a modal filter (hotkey 1)"
-      >
-        <img
-          src={`${import.meta.env.BASE_URL}/filters/${$filterType}_icon.gif`}
-          alt="Add a modal filter"
-        />
-      </button>
-      <button
-        on:click={() => (action = { kind: "freehand-filters" })}
-        disabled={action.kind == "freehand-filters"}
-        class:active={action.kind == "freehand-filters"}
-        class:outline={action.kind != "freehand-filters"}
-        data-tooltip="Add many modal filters along a line (hotkey 2)"
-      >
-        <img
-          src={`${import.meta.env.BASE_URL}/filters/select_freehand.png`}
-          alt="Add many modal filters along a line"
-        />
-      </button>
-      <button
-        on:click={() => (action = { kind: "oneway" })}
-        disabled={action.kind == "oneway"}
-        class:active={action.kind == "oneway"}
-        class:outline={action.kind != "oneway"}
-        data-tooltip="Toggle one-way (hotkey 3)"
-      >
-        <!-- 
+    <div
+      class="tool-palette"
+      style="height: 54px; display: flex; justify-content: space-between; gap: 32px;"
```

Comment:
On my end, the fixed height of the panel jumps around for some of the tools. The screenshot region below is the same, but you can see "Map style" jumps:
![image](https://github.com/user-attachments/assets/f3f2b1c1-5ba7-4741-b69d-30e6eb2cf347)
vs
![image](https://github.com/user-attachments/assets/45247ad6-36f7-42af-85b6-6cecc5b09480)

This could be resolution or something else. Happy to look into this later.

Reply:
It's the same on my machine. I actually didn't intend for there to be *no* difference in height, just to reserve enough space so the area was "recognizable" as the tab content area.

If we wanted no movement whatsover, we'd need to find our longest content at our narrowest width. Do you want me to make that change?